project4/project/ebs/templates/ebs/update_blog.html

```diff
@@ -34,31 +34,31 @@ <h3 class="header">EDIT BLOG</h3>
             </div>
         </div>
         <div class="row">
-            <div class="form-group col-md-6">
+            <div id ="attached-row" class="form-group col-md-6">
                 {{blogform.image.errors}}
-                <label class="control-label">Attachments</label><br>
+                <label id="attached" class="control-label">Attachments</label>
                     {{blogfileform.attachments}}
                     {{blogfileform.image1}}
                     {{blogfileform.image2}}
             </div>
         </div>
         <div class="row">
-            <div class="upload col-md-6">
+            <div id="upload-row" class="upload form-group col-md-6">
         {% if attachments_value %}
                 <label id="label-uploaded">Uploaded Files</label>
-                <div id="file1">{{attachments_value}}
-                    <button id="button1" type="button" value="{{attachments_id}}"><i class="fa fa-trash-o" aria-hidden="true"></i></button>
-                </div>
+                <p id="file1">{{attachments_value}}
```

Comment:
- add some class to p tag and use "word-wrap:break-word;" to avoid long filename overlapping on the delete icon.
 - on large filenames delete button should be aligned to top.
 - use padding and margin to add space between filename and button.